src/lib/utils/SynapseClient.ts

```diff
@@ -1169,7 +1171,8 @@ export const getPresignedUrlForWikiAttachment = (
 }
 
 export const isInSynapseExperimentalMode = (): boolean => {
-  return cookies.get(SynapseConstants.EXPERIMENTAL_MODE_COOKIE)
+  // bang bang, you're a boolean!
+  return !!cookies.get(SynapseConstants.EXPERIMENTAL_MODE_COOKIE)
```

Comment:
This was sometimes causing an error when in Experimental Mode (Entity Badge Item for example, expected a boolean but it's an object).